### PR TITLE
fix: Fixed passing `positional only` arguments in few functions

### LINKED ISSUE #27244: `Position only` arguments are passed as key-word args
In some files while calling some of the functions, `position only arguments` (those to the left of `/` in the function signature) are passed as `keyword arguments`.
https://github.com/unifyai/ivy/blob/3678f56e2d60be5ab5065905a6eb2f4eb008c27c/ivy/functional/frontends/jax/numpy/statistical.py#L36
https://github.com/unifyai/ivy/blob/3678f56e2d60be5ab5065905a6eb2f4eb008c27c/ivy/functional/frontends/mindspore/ops/function/nn_func.py#L339
https://github.com/unifyai/ivy/blob/3678f56e2d60be5ab5065905a6eb2f4eb008c27c/ivy/functional/frontends/torch/nn/functional/non_linear_activation_functions.py#L70
https://github.com/unifyai/ivy/blob/3678f56e2d60be5ab5065905a6eb2f4eb008c27c/ivy/functional/frontends/mxnet/numpy/random.py#L65
https://github.com/unifyai/ivy/blob/3678f56e2d60be5ab5065905a6eb2f4eb008c27c/ivy/functional/frontends/scipy/spatial/distance.py#L32

These will result in `TypeError: got some positional-only arguments passed as keyword arguments`
So, they must be corrected.

### PR DESCRIPTION
**Pull request by Sai-Suraj-27**
# PR Description
In some files while calling some of the functions, `position only arguments` (those to the left of `/` in the function signature) are passed as `keyword arguments`.
https://github.com/unifyai/ivy/blob/3678f56e2d60be5ab5065905a6eb2f4eb008c27c/ivy/functional/frontends/jax/numpy/statistical.py#L36
https://github.com/unifyai/ivy/blob/3678f56e2d60be5ab5065905a6eb2f4eb008c27c/ivy/functional/frontends/mindspore/ops/function/nn_func.py#L339
https://github.com/unifyai/ivy/blob/3678f56e2d60be5ab5065905a6eb2f4eb008c27c/ivy/functional/frontends/torch/nn/functional/non_linear_activation_functions.py#L70
https://github.com/unifyai/ivy/blob/3678f56e2d60be5ab5065905a6eb2f4eb008c27c/ivy/functional/frontends/mxnet/numpy/random.py#L65
https://github.com/unifyai/ivy/blob/3678f56e2d60be5ab5065905a6eb2f4eb008c27c/ivy/functional/frontends/scipy/spatial/distance.py#L32

## Related Issue
Closes #27244

## Checklist

- [ ] Did you add a function?
- [ ] Did you add the tests?
- [ ] Did you run your tests and are your tests passing?
- [x] Did pre-commit not fail on any check?
- [ ] Did you follow the steps we provided?

### Socials
https://twitter.com/Sai_Suraj_27

### CODE REVIEW

**Review by KareemMAX — APPROVED** [review]
LGTM! Maybe @vedpatwardhan can be a second pair of eyes before we merge.

**Review by vedpatwardhan — APPROVED** [review]
lgtm! Feel free to merge, thanks @Sai-Suraj-27 @KareemMAX 😄

**Pull request merged**
